Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ruby:2.6.6
+
+# install java & set path
+RUN apt-get update && \
+    apt-get -y install -y sudo
+
+RUN apt-get install -y openjdk-11-jdk
+RUN sudo update-alternatives --config java
+
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk-amd64/
+RUN export JAVA_HOME
+RUN javac -version
+
+# install ant
+RUN apt update
+RUN apt -y install ant
+RUN ant -version
+
+# install antlr
+WORKDIR /usr/local/lib
```

Comment:
This seems to be about linux installations only.  Is this the same instructions for a standard install for (cyg)win?

Reply:
From my understanding, the containers are all linux-based.